src/controllers/index.js

```diff
@@ -0,0 +1,7 @@
+const ConfigService = require('../services/configService.js')
+const create = require('./handleArguments')
```

Comment:
I think both names should align here. I am also thorn on the module itself being called handleArguments, but I might be nitpicking.
```suggestion
const handleArguments = require('./handleArguments')
```